src/lsdb/io/to_hats.py

```diff
@@ -90,13 +88,9 @@ def to_hats(
         catalog_name if catalog_name else catalog.hc_structure.catalog_name,
         total_rows=sum(pi[0] for pi in partition_info.values()),
     )
-    hc.io.write_catalog_info(catalog_base_dir=base_catalog_path, dataset_info=new_hc_structure.catalog_info)
+    new_hc_structure.catalog_info.to_properties_file(base_catalog_path)
     # Save provenance info
-    hc.io.write_metadata.write_provenance_info(
-        catalog_base_dir=base_catalog_path,
-        dataset_info=new_hc_structure.catalog_info,
-        tool_args=_get_provenance_info(new_hc_structure),
-    )
+    ## TODO - optionally write out arguments file
```

Comment:
Push to another issue?

Reply:
Removed the TODO. On further reflection, I don't think we have any additional fields we'd want to write to such a file.